Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/08-RecordMeeting/ImplementingTimer-04-code-0016.swift

```diff
@@ -24,6 +24,7 @@ final class RecordMeetingTests: XCTestCase {
       $0.continuousClock = clock
       $0.date.now = Date(timeIntervalSince1970: 1234567890)
       $0.uuid = .incrementing
+      $0.dismiss = DismissEffect {}
```

Comment:
Applies the most basic override before we give it something better in the next step, per

> Override the `dismiss` dependency in the `withDependencies` trailer closure.

Reply:
Just curious was there actually a `dismiss`-related test failure in your runthrough? There used to be, but these days the test store automatically tears down effects when `dismiss()` is called by the reducer.